quic/s2n-quic-sim/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "s2n-quic-sim"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
This one is also unpublished so it shouldn't need a version bump. Would be good to add a note similar to the other unpublished crates

Reply:
> Would be good to add a note similar to the other unpublished crates

Yeah that would probably be good

Reply:
added